README.md

```diff
@@ -393,7 +394,17 @@ acquireToken method without activity supports dialog prompt.
 
 ### Encryption
 
-ADAL encrypts the tokens and store in SharedPreferences by default. You can look at the StorageHelper class to see the details. Android introduced AndroidKeyStore for 4.3(API18) secure storage of private keys. ADAL uses that for API18 and above. If you want to use ADAL for lower SDK versions, you need to provide secret key at AuthenticationSettings.INSTANCE.setSecretKey
+ADAL encrypts the tokens and store in SharedPreferences by default. You can look at the StorageHelper class to see the details. Android introduced AndroidKeyStore for 4.3(API18) secure storage of private keys. ADAL uses that for API18 and above. If you want to use ADAL for lower SDK versions, you need to **provide secret key at AuthenticationSettings.INSTANCE.setSecretKey**
+
+```Java
+	SecretKeyFactory keyFactory = SecretKeyFactory
+		.getInstance("PBEWithSHA256And256BitAES-CBC-BC");
```

Comment:
does ADAL require the algorithm to be "PBEWithSHA256And256BitAES-CBC-BC"? If not, then lets not put something that may be perceived as a requirement. Someone may want to use SHA512 or SHA384.